src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+  const totalCost = days * cost;
+
+  if (days >= 7) {
+    return totalCost - discountBig;
+  } else if (days < 7 && days >= 3) {
+    return totalCost - discountSmall;
+  } else {
+    return totalCost;
+  }
```

Comment:
When you use return you don't need `else` statement. 
```suggestion
  if (days >= 7) {
    return totalCost - discountBig;
  } 
  
  if (days < 7 && days >= 3) {
    return totalCost - discountSmall;
  }
  
  return totalCost;
```